docs/workflow/trimming/feature-switches.md

```diff
@@ -15,6 +15,7 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 | InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |
 | UseSystemResourceKeys | System.Resources.UseSystemResourceKeys |  Any localizable resources for system assemblies is trimmed when set to true |
 | HttpActivityPropagationSupport | System.Net.Http.EnableActivityPropagation | Any dependency related to diagnostics support for System.Net.Http is trimmed when set to false |
+| StartupHookSupport | System.StartupHookProvider.IsSupported | Any StartupHook related logic is trimmed when set to false |
```

Comment:
`StartupHookSupport ` => I assume you plan on adding this property to the SDK as well?

https://github.com/dotnet/sdk/blob/f48d1dd517cda52bfc7345fb2d16477476a33377/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L340-L416

Reply:
Yes. I opened this issue already: https://github.com/dotnet/sdk/issues/14385 I'll put a PR when this one is merged.